apps/wallet-dashboard/components/VirtualList.tsx

```diff
@@ -51,17 +51,10 @@ function VirtualList<T>({
         if (lastItem.index >= items.length - 1 && hasNextPage && !isFetchingNextPage) {
             fetchNextPage();
         }
-    }, [
-        hasNextPage,
-        fetchNextPage,
-        items.length,
-        isFetchingNextPage,
-        virtualizer,
-        virtualizer.getVirtualItems(),
```

Comment:
why has this been removed from the useEffect?
Its still needed to get the last item